dist/api_err.py

```diff
@@ -83,32 +132,31 @@ def __init__(self, name, value, desc, long_desc=None, **flags):
         setting.'''),
 ]
 
+sub_errors = [
+    Error('WT_NONE', -32000,
+    'last API call was successful', '''
+    This is the default sub-level error code that should be used when there is no
+    sub-level error to pair with an error. It indicates that no further context
+    exists or is necessary.'''),
+    Error('WT_COMPACTION_ALREADY_RUNNING', -32001,
```

Comment:
While this is correct, please can you make the indenting similar to the 'error' section, as that will make the code easier to read.

Reply:
Fixed

Reply:
Thanks!